code/__HELPERS/logging.dm

```diff
@@ -118,15 +118,26 @@
  * @param amount The amount of units transferred
  * @param reagent_names List of reagent names to log
  */
-/proc/log_reagents(var/mob/user, var/source, var/target, var/amount, var/list/reagent_names)
-	if (amount == 0)
+/proc/log_reagent_transfer(var/mob/user, var/atom/source, var/atom/target, var/amount_desired)
+	if(amount_desired == 0)
 		return
-
-	if (reagent_names && reagent_names.len > 0)
-		var/reagent_text = "<span class='danger'>[english_list(reagent_names)]</span>"
-		add_gamelogs(user, "added [amount]u (inc. [reagent_text]) to \a [target] with \the [source]", admin = TRUE, tp_link = TRUE)
-	else
-		add_gamelogs(user, "added [amount]u to \a [target] with \the [source]", admin = TRUE, tp_link = FALSE)
+	if(isobj(target)) //Wouldn't it be nice to just use : sometimes?
```

Comment:
Not in this case, at least

Also, this should be `istype(target, /obj)` because, as stupid as this sounds, `isobj()` returns true for any `/atom/movable` except mobs, not just `/obj`, and this is an `/obj`-level variable